README.md

```diff
@@ -64,6 +64,11 @@ Sometimes this is just a more practical and quick way than doing things properly
     chmod +x miniserve-osx
     ./miniserve-osx
 
+Alternatively install view [Homebrew](https://brew.sh/).
```

Comment:
Probably a typo. I suggest: `Alternatively install with`

Reply:
Apologies, you are correct.

Reply:
Cool, I'll merge once you change it.

Reply:
Done!